services/src/main/java/org/exoplatform/addons/gamification/service/mapper/GamificationActionsHistoryMapper.java

```diff
@@ -0,0 +1,44 @@
+package org.exoplatform.addons.gamification.service.mapper;
+
+import org.exoplatform.addons.gamification.IdentityType;
+import org.exoplatform.addons.gamification.Utils.Utils;
+import org.exoplatform.addons.gamification.entities.domain.effective.GamificationActionsHistory;
+import org.exoplatform.addons.gamification.service.dto.configuration.GamificationActionsHistoryDTO;
+
+public class GamificationActionsHistoryMapper {
+
+  private DomainMapper domainMapper;
+
+  private RuleMapper   ruleMapper;
```

Comment:
change method public static in DomainMapper & RuleMapper and delete this from the file